accounts-db/src/accounts_db.rs

```diff
@@ -2399,13 +2399,13 @@ impl<'a> AppendVecScan for ScanState<'a> {
             } else if self.config.check_hash && computed_hash != loaded_hash {
                 info!(
                     "hash mismatch found: computed: {}, loaded: {}, pubkey: {}",
-                    computed_hash, loaded_hash, pubkey
+                    computed_hash.0, loaded_hash.0, pubkey
```

Comment:
&computed_hash?

Reply:
I was thinking you implemented `Borrow`, I think. But if you don't specify what you're borrowing as, it likely will be &AccountsHash instead of calling the borrow &Hash.

I think I'd prefer display or debug fmt than .0. But, this isn't that important. Probably fine to leave as-is.